pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -228,14 +228,22 @@ message PodSandboxConfig {
     repeated PortMapping port_mappings = 5;
     // Key-value pairs that may be used to scope and select individual resources.
     map<string, string> labels = 6;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata. There are a few features are
-    // driven by annotations, Runtimes could support them optionally:
+    // Unstructured key value map that may be set by external tools to store
+    // and retrieve arbitrary metadata. Annotations MUST NOT be altered or in
+    // any way processed by the runtime.
+    //
+    // In order to preserve a well-defined interface between the kubelet and
+    // the container runtime, annotations SHOULD NOT influence runtime behaviour.
```

Comment:
One use case I could see for annotations being passed to the runtime is, if I were developing a CRI implementation that had some experimental features - Kubernetes might not be ready to take those features into the API (even as alpha fields), but I could still implement them through runtime-specific annotations.